great_expectations/data_context/migrator/cloud_migrator.py

```diff
@@ -143,81 +150,145 @@ def migrate_validation_result(
+        response = self._post_to_cloud_backend(
+            resource_name="migration",
+            resource_type="migration",
+            attributes_key="bundle",
+            attributes_value=serialized_bundle,
+        )
+        print(response)
+        # TODO: Handle success/failure cases
+
+    def _prepare_validation_results(self, serialized_bundle: dict) -> List[dict]:
+        print("Preparing validation results (step 3/4")
+        return serialized_bundle.pop("validation_results")
+
+    def _send_validation_results(
+        self, serialized_validation_results: List[dict]
+    ) -> None:
+        # 20220928 - Chetan - We want to use the static lookup tables in GeCloudStoreBackend
+        # to ensure the appropriate URL and payload shape. This logic should be moved to
+        # a more central location.
+        resource_type = GeCloudRESTResource.EXPECTATION_VALIDATION_RESULT
+        resource_name = GeCloudStoreBackend.RESOURCE_PLURALITY_LOOKUP_DICT[
+            resource_type
+        ]
+        attributes_key = GeCloudStoreBackend.PAYLOAD_ATTRIBUTES_KEYS[resource_type]
+
+        print("Sending validation results (step 4/4)")
+        for i, validation_result in enumerate(serialized_validation_results):
+            response = self._post_to_cloud_backend(
+                resource_name=resource_name,
+                resource_type=resource_type,
+                attributes_key=attributes_key,
+                attributes_value=validation_result,
+            )
+            print(response)
+            # TODO: Handle success/failure cases
+
+    def _post_to_cloud_backend(
         self,
-        configuration_bundle: ConfigurationBundle,
-        serializer: ConfigurationBundleJsonSerializer,
-    ) -> AnyPayload:
+        resource_name: str,
+        resource_type: str,
+        attributes_key: str,
+        attributes_value: dict,
+    ) -> MigrationResponse:
         url = construct_url(
             base_url=self._ge_cloud_base_url,
             organization_id=self._ge_cloud_organization_id,
-            resource_name="migration",
+            resource_name=resource_name,
         )
-
-        serialized_bundle = serializer.serialize(configuration_bundle)
         data = construct_json_payload(
-            resource_type="migration",
+            resource_type=resource_type,
             organization_id=self._ge_cloud_organization_id,
-            attributes_key="bundle",
-            attributes_value=serialized_bundle,
+            attributes_key=attributes_key,
+            attributes_value=attributes_value,
         )
+        response = self._session.post(url, json=data)
+        return self._parse_cloud_response(response=response)
 
-        try:
-            response = self._session.post(url, json=data)
-            response.raise_for_status()
-            return response.json()
+    def _parse_cloud_response(self, response: requests.Response) -> MigrationResponse:
+        success = response.ok
```

Comment:
You might want to consider any `2XX` response a success.
This will only consider a `200` code a success but count `201`, `202`, `204` etc as `sucess = False` which seems wrong.

Reply:
https://requests.readthedocs.io/en/latest/api/?highlight=OK#requests.Response.ok shows that it works for all codes less that 400